src/layouts/footer/FooterLayoutCss.scss

```diff
@@ -0,0 +1,100 @@
+$primary-color: #fff;
+$link-color: white;
+$link-hover-color: pink;
+
+.footer {
+	background-color: #006400;
+	font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
```

Comment:
Font faces are already declared in `index.css`:
```css
:root {
	font-family: 'Montserrat', 'Raleway', Helvetica, Arial, sans-serif;

	--console-font: 'Inconsolata', monospace;
	--handwriting-font: 'Caveat', cursive;
	--styled-font: 'Lobster', cursive;
	--fancy-3d-font: 'Nabla', cursive;
	text-align: center;
	font-size: 16px;
}
```
If any more are required add them there.